tests/PySys/log_request_end_to_end/log_request_num_lines/run.py

```diff
@@ -4,6 +4,8 @@
 import subprocess
 import requests
 from retry import retry
+from create_test_logs import create_example_logs
```

Comment:
`create_test_logs` is one weird name for a python module. How about `test_log_generator`?

Reply:
yes, renamed it